portfolio/src/main/webapp/style.css

```diff
@@ -55,6 +55,25 @@ header img {
 }
 
 
+.comment {
+       border: thin solid white;
```

Comment:
Can you add a screenshot in the pull request description? I'm curious to see what this looks like!

Reply:
done!